sql-statements/sql-statement-import-into.md

```diff
@@ -10,6 +10,8 @@ The `IMPORT INTO` statement lets you import data to TiDB via the [Physical Impor
 - `IMPORT INTO ... FROM FILE`: imports data files in formats such as `CSV`, `SQL`, and `PARQUET` into an empty table in TiDB.
 - `IMPORT INTO ... FROM SELECT`: imports the query result of a `SELECT` statement into an empty table in TiDB. You can also use it to import historical data queried with [`AS OF TIMESTAMP`](/as-of-timestamp.md).
 
+The [TiDB Distributed eXecution Framework (DXF)](/tidb-distributed-execution-framework.md) can be used to speedup the operation of this statement.
```

Comment:
The details about DXF are already provided in L169, so perhaps we can skip adding this line here?

![image](https://github.com/pingcap/docs/assets/79440533/17f4b768-4aa6-4a18-abb8-cb7b4d585918)